pkg/api/server/register_manifest4.go

```diff
@@ -0,0 +1,181 @@
+package server
+
+import (
+	"net/http"
+
+	"github.com/containers/podman/v3/pkg/api/handlers/libpod"
+	"github.com/gorilla/mux"
+)
+
+func (s *APIServer) registerManifest4Handlers(r *mux.Router) error {
+	// swagger:operation POST /libpod/manifests/create manifests ManifestCreateLibpod
+	// ---
+	// summary: Create
+	// description: |
+	//   Create a manifest list
+	//
+	//   As of v4.0.0
+	// produces:
+	// - application/json
+	// parameters:
+	// - in: query
+	//   name: name
+	//   type: string
+	//   description: manifest list or index name to create
+	//   required: true
+	// - in: query
+	//   name: image
+	//   type: string
+	//   description: |
+	//     One or more names of an image or a manifest list. Repeat parameter as needed.
+	// - in: query
+	//   name: all
+	//   type: boolean
+	//   description: add all contents if given list
```

Comment:
This isn't completely clear. List as in manifest list? So basically, if there's a manifest list given in `image`, take all the contents of said list?

What happens if this isn't given and there's a manifest list in `image`?